frontend/src/App.tsx

```diff
@@ -26,19 +27,21 @@ const App = () => {
   });
   // ***PURELY FOR EXAMPLE***
 
-  return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<UserInterface />}>
-          <Route path="home" element={<Home />} />
-          <Route path="explore" element={<h1>Explore</h1>} />
-          <Route path="products" element={<h1>Products</h1>} />
-          <Route path="mentors" element={<h1>Mentors</h1>} />
+
+	return (
+		<Router>
+			<Routes>
+				<Route path="/" element={<UserInterface />}>
+					<Route path="home" element={<Home />} />
+					<Route path="explore" element={<h1>Explore</h1>} />
+					<Route path="products" element={<h1>Products</h1>} />
+					<Route path="mentors" element={<h1>Mentors</h1>} />
+					<Route path="profile" element={<Profile />} />
```

Comment:
Change your formatter to use smaller tab width

Reply:
got it. 2 space for JS